src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -569,10 +581,9 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
       // Running and Aborting have explicit handlers for Running and no pending writes, so either we are not in those
       // states or the specified key is no longer Running and/or has pending writes.
       stay()
-    case Event(PersistenceCompleted(callKey, status, _), data) =>
-      logger.debug(s"Got whenUnhandled PersistenceCompleted(${callKey.tag}): $status")
-      val updatedData = data.removePersisting(callKey, status)
-      stay using updatedData
+    case Event(m: PersistenceMessage, data) =>
+      logger.debug(s"Got whenUnhandled message: $m")
```

Comment:
Is this debug necessary? It looks like this is really the expected path for Persistence message handling?